src/anndata/_core/views.py

```diff
@@ -208,6 +208,19 @@ def keys(self) -> KeysView[str]:
         # it’s a structured array
         return self.dtype.names
 
+    # New Feature to convert DaskArrayView to DaskArray to write easily (see https://github.com/scverse/anndata/issues/2022)
+    def to_dask_array(self) -> DaskArray:
```

Comment:
Hi @Flakes342 thanks for the PR, sorry I missed it - I opened a PR and then saw your PR in the issue.  Have a look at the PR I opened for how this can be done simpler in the future! Thanks for the contribution, though, feel free to contribute in the future and hopefully I notice haha